package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "brainstem-redux",
-  "version": "0.1.3",
+  "version": "0.1.3-babel7",
   "description": "Client-side Brainstem store in Redux",
   "main": "bin/index.js",
```

Comment:
@juanca I'm trying to understand what exactly happens when you do
`import brainstemRedux from 'brainstem-redux';` ... what's the file that's pointed to?
Reading 
https://docs.npmjs.com/files/package.json#main


I can't see how all this is even working. We don't appear have a `bind/index`, nor do we use the `bin: ` key to set that up to something that exists like described:
https://docs.npmjs.com/files/package.json#bin

This comment corresponds with my other comment at:

https://github.com/mavenlink/mavenlink/pull/16440/files#diff-f1d7259a1dcdd24ba6419ac686c53f09R1

Reply:
Ok, I found it:

https://github.com/mavenlink/brainstem-redux/blob/master/webpack.compile.config.js#L13

Ok, so disregard above in terms of how bin/index.js happens. However, I think you're importing these incorrectly on the other PR. You'll want to do `import * as brainstem-redux...` since api.js is now a bunch of named exports essentially.

Reply:
Yeah, the other PR is not pointing to the latest change set from this PR. This PR published a prerelease version a bit ago. I will publish another later and see how it fares.